styles/variables/_timing.scss

```diff
@@ -17,3 +17,6 @@
 ////////////////////////////////////////////////////////////////////////////
 
 $fast: 100ms;
+$medium: 300ms;
```

Comment:
The word "medium" doesn't entail a duration (like fast and slow) and since this defines a global SCSS variable, we might wan't to choose words which are more precise?

Reply:
Maybe prefix all durations with `duration`, e.g. `$duration-fast` etc?

Reply:
Well it's actually not used anymore, so removed for now.